Sources/Tracking/Background Queue/QueueRunRequest.swift

```diff
@@ -11,14 +11,22 @@ public class CioQueueRunRequest: QueueRunRequest {
     private let storage: QueueStorage
     private let requestManager: QueueRequestManager
     private let logger: Logger
+    private let queryRunner: QueueQueryRunner
 
     private let shortTaskId: (String) -> String = { $0[0 ..< 5] }
 
-    init(runner: QueueRunner, storage: QueueStorage, requestManger: QueueRequestManager, logger: Logger) {
+    init(
+        runner: QueueRunner,
+        storage: QueueStorage,
+        requestManger: QueueRequestManager,
```

Comment:
very festive but 
```suggestion
        requestManager: QueueRequestManager,
```
is probably better

Reply:
fixed